examples/redux/ngrx-actions/app/index.ts

```diff
@@ -1,11 +1,23 @@
-import { bootstrap } from '@angular/platform-browser-dynamic';
-import { provide } from '@angular/core';
-import { provideStore } from '@ngrx/store';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { StoreModule } from '@ngrx/store';
 
 import { SimpleNgrx } from './containers/app-container';
+import { Counter } from './components/counter-component';
 import { counterReducer } from './reducers/counter-reducer';
 
-bootstrap(SimpleNgrx, [
-  provideStore({ counter: counterReducer })
-]);
-
+@NgModule({
```

Comment:
This specific file is for an Ngrx example, the configuration for it is done in the import itself

Point taken for the other Redux examples though.
I originally attempted store configuration inside of the AppModule constructor but kept getting this exception `Error: Can't resolve all parameters for AppModule: (?).`
I don't know if this is a bug, or if Angular no longer supports DI on a module class.

Take a look at this plnkr - maybe I'm missing something obvious.
https://plnkr.co/edit/pXBAH11x14MOUz2U9w1r?p=preview